phpBB/language/en/acp/common.php

```diff
@@ -88,6 +88,11 @@
 	'ACP_EXTENSION_GROUPS'		=> 'Manage attachment extension groups',
 	'ACP_EXTENSION_MANAGEMENT'	=> 'Extension management',
 	'ACP_EXTENSIONS'			=> 'Manage extensions',
+	'ACP_EXTENSIONS_GALLERY'	=> 'Extensions gallery',
```

Comment:
that implies $$
My ideas:
Catalog
Depot
Database
Collection
Exhibit
Browse Extensions